app/src/main/res/layout/profile_chooser_fragment.xml

```diff
@@ -11,24 +11,27 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/profileChooserBackground">
+
     <TextView
       android:id="@+id/profile_select_text"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      app:layout_constraintTop_toTopOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
+      android:layout_marginStart="36dp"
```

Comment:
Add `android:layout_marginEnd="36dp"` for RTL support

Reply:
done.